### PR TITLE
bloop: fix hashes for the v1.4.13

### PR DESCRIPTION
**Pull request by svalaskevicius**
###### Description of changes

Original hashes resulted in bloop 1.4.12 being installed.

###### Things done

1. Replaced hashes with random strings
2. Ran the local install
3. Updated hashes to what was given

Result: bloop 1.4.13

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by svalaskevicius** [comment]
@Tomahna, @kubukoz, please can you review? 

Thanks!

**Comment by svalaskevicius** [comment]
I expect there is likely an issue left with the hash for Darwin, but I don't have such environment to verify..

**Comment by kubukoz** [comment]
I'm actually not able to build this on Darwin at all...

<details>
<summary>log</summary>


```
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
no Makefile, doing nothing
@nix { "action": "setPhase", "phase": "installPhase" }
installing
Downloaded 1 missing file(s) / 62
Downloaded 2 missing file(s) / 62
Downloaded 3 missing file(s) / 62
Downloaded 4 missing file(s) / 62
Downloaded 5 missing file(s) / 62
Downloaded 6 missing file(s) / 62
Downloaded 7 missing file(s) / 62
Downloaded 8 missing file(s) / 62
Downloaded 9 missing file(s) / 62
Downloaded 10 missing file(s) / 62
Downloaded 11 missing file(s) / 62
Downloaded 12 missing file(s) / 62
Downloaded 13 missing file(s) / 62
Downloaded 14 missing file(s) / 62
Downloaded 15 missing file(s) / 62
Downloaded 16 missing file(s) / 62
Downloaded 17 missing file(s) / 62
Downloaded 18 missing file(s) / 62
Downloaded 19 missing file(s) / 62
Downloaded 20 missing file(s) / 62
Downloaded 21 missing file(s) / 62
Downloaded 22 missing file(s) / 62
Downloaded 23 missing file(s) / 62
Downloaded 24 missing file(s) / 62
Downloaded 25 missing file(s) / 62
Downloaded 26 missing file(s) / 62
Downloaded 27 missing file(s) / 62
Downloaded 28 missing file(s) / 62
Downloaded 29 missing file(s) / 62
Downloaded 30 missing file(s) / 62
Downloaded 31 missing file(s) / 62
Downloaded 32 missing file(s) / 62
Downloaded 33 missing file(s) / 62
Downloaded 34 missing file(s) / 62
Downloaded 35 missing file(s) / 62
Downloaded 36 missing file(s) / 62
Downloaded 37 missing file(s) / 62
Downloaded 38 missing file(s) / 62
Downloaded 39 missing file(s) / 62
Downloaded 40 missing file(s) / 62
Downloaded 41 missing file(s) / 62
Downloaded 42 missing file(s) / 62
Downloaded 43 missing file(s) / 62
Downloaded 44 missing file(s) / 62
Downloaded 45 missing file(s) / 62
Downloaded 46 missing file(s) / 62
Downloaded 47 missing file(s) / 62
Downloaded 48 missing file(s) / 62
Downloaded 49 missing file(s) / 62
Downloaded 50 missing file(s) / 62
Downloaded 51 missing file(s) / 62
Downloaded 52 missing file(s) / 62
Downloaded 53 missing file(s) / 62
Downloaded 54 missing file(s) / 62
Downloaded 55 missing file(s) / 62
Downloaded 56 missing file(s) / 62
Downloaded 57 missing file(s) / 62
Downloaded 58 missing file(s) / 62
Downloaded 59 missing file(s) / 62
Downloaded 60 missing file(s) / 62
Downloaded 61 missing file(s) / 62
Downloaded 62 missing file(s) / 62
Exception in thread "main" java.nio.file.FileSystemException: /var/empty/Library: Operation not permitted
        at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:100)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
        at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:398)
        at java.base/java.nio.file.Files.createDirectory(Files.java:700)
        at java.base/java.nio.file.Files.createAndCheckIsDirectory(Files.java:807)
        at java.base/java.nio.file.Files.createDirectories(Files.java:793)
        at coursier.paths.Util.createDirectories(Util.java:81)
        at coursier.paths.CoursierPaths.computeCacheDirectory(CoursierPaths.java:62)
        at coursier.paths.CoursierPaths.computeArchiveCacheDirectory(CoursierPaths.java:43)
        at coursier.paths.CoursierPaths.archiveCacheDirectory(CoursierPaths.java:84)
        at coursier.paths.CachePath.defaultArchiveCacheDirectory(CachePath.java:109)
        at coursier.cache.CacheDefaults$.archiveCacheLocation$lzycompute(CacheDefaults.scala:18)
        at coursier.cache.CacheDefaults$.archiveCacheLocation(CacheDefaults.scala:18)
        at coursier.cache.ArchiveCache$.apply(ArchiveCache.scala:141)
        at coursier.install.InstallDir$.apply(InstallDir.scala:56)
        at coursier.cli.install.SharedInstallParams.installDir(SharedInstallParams.scala:24)
        at coursier.cli.install.Install$.run(Install.scala:53)
        at coursier.cli.install.Install$.run(Install.scala:21)
        at caseapp.core.app.CaseApp.main(CaseApp.scala:149)
        at caseapp.core.app.CommandsEntryPoint.main(CommandsEntryPoint.scala:109)
        at coursier.cli.Coursier$.main(Coursier.scala:115)
        at coursier.cli.Coursier.main(Coursier.scala)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at coursier.bootstrap.launcher.a.a(Unknown Source)
        at coursier.bootstrap.launcher.Launcher.main(Unknown Source)
```

</details>

File: pkgs/development/tools/build-managers/bloop/default.nix [hunk 1]
```diff
@@ -60,7 +60,7 @@ stdenv.mkDerivation rec {
 
     outputHashMode = "recursive";
     outputHashAlgo = "sha256";
-    outputHash = if stdenv.isLinux && stdenv.isx86_64 then "sha256-jqcecAM51qEDmTim2VBNm8IO8wQmwU19R57Zk4pxwSA="
+    outputHash = if stdenv.isLinux && stdenv.isx86_64 then "sha256-AiF/ih15Jd0WuDP/0vU0vdaSo3FGjWXos+hNVBayFz4="
     else if stdenv.isDarwin && stdenv.isx86_64 then "sha256-WJytRIbsygi4zoIVfkJmzWyFe2p8mQuT4DDO5KDKopY="
```
Comment (kubukoz) [inline pkgs/development/tools/build-managers/bloop/default.nix:64]:
```suggestion
    else if stdenv.isDarwin && stdenv.isx86_64 then "sha256-LD23YpcNhWfioGDMqb1plqLy87ZHzT0zvIyc4O4WP5g="
```

**Review by kubukoz — CHANGES_REQUESTED** [review]

**Comment by svalaskevicius** [comment]
applied, thanks!

**Review by kubukoz — APPROVED** [review]

**Review by Tomahna — APPROVED** [review]
Compiled and tested on nixos :+1: .

**Comment by kubukoz** [comment]
@turboMaCk can you merge please?

**Pull request merged**